playground/webpack.config.js

```diff
@@ -20,6 +21,9 @@ module.exports = (env, argv) => {
       sourceMapFilename: '[name].map',
       publicPath: '',
     },
+    node: {
+      global: false,
```

Comment:
tell webpack to not include the [polyfill](https://github.com/webpack/webpack/blob/911ec1aa67011e25aa1449610f5b0b557edd5459/lib/runtime/GlobalRuntimeModule.js#L20-L42)